### PR TITLE
Result page

### PR DESCRIPTION
**Pull request by kirti-d**
front end for resultPage of webApp 
tech used :- 
 html and css

### CODE REVIEW

File: src/main/webapp/result.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <title>Error analyzer</title>
+
+    <!-- add css -->
+    <link rel="stylesheet" href="result.css">
+    <!-- add fontawesome icons -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" 
+    integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+</head>
+
+<body onload="changePage()">
```
Comment (nishantnahata) [inline src/main/webapp/result.html:32]: js file was submitted already?

File: src/main/webapp/result.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <!-- header -->
+
+    <!-- show result -->
+    <div>
+        <div id="listingTable"></div>
+    </div>
+    <!-- navigate between pages -->
+    <footer class="backgroundColor text-center">
+        <div id="navigation">
+            <a href="javascript:prevPage()" id="btnPrev">Prev</a>
```
Comment (nishantnahata) [inline src/main/webapp/result.html:64]: Didn't know this works :p
I used to add hyperlink style in css to buttons for this.

File: src/main/webapp/result.html (hunk 1 above)
Comment (kirti-d) [reply src/main/webapp/result.html:32]: its there in pagination branch

File: src/test/java/com/google/error_analyzer/backend/DocCountServletTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    @Before
+    public void setUp() {
+        docCount = new DocCountServlet();
+        ReflectionTestUtils.setField(docCount, "logDao", logDao);
+        MockitoAnnotations.initMocks(this);
+    }
+
+    // test dopost
+    @Test
+    public void returnCount() throws Exception {
```
Comment (nishantnahata) [inline src/test/java/com/google/error_analyzer/backend/DocCountServletTest.java:68]: {function under test}_returnCount

File: src/main/java/com/google/error_analyzer/backend/LogDao.java [hunk 4]
```diff
@@ -85,6 +87,17 @@
         return ImmutableList.copyOf(Arrays.asList(searchHits));
     }
 
+    // returns no of documents in an index
+    @Override 
+    public long getDocCount (String index) throws IOException {
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/backend/LogDao.java:92]: Optional:

getDocumentCount

In variable names, feel free to use the complete words.

File: src/main/java/com/google/error_analyzer/servlets/DocCountServlet.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+
+@WebServlet("/getCount")
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/servlets/DocCountServlet.java:29]: Optional:
getDocumentCount

File: src/main/java/com/google/error_analyzer/servlets/DocCountServlet.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+public class DocCountServlet extends HttpServlet {
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/servlets/DocCountServlet.java:30]: Optional:
DocumentCountServlet

**Pull request merged**